app/gui2/src/App.vue

```diff
@@ -56,7 +58,7 @@ const appConfig = computed(() => {
   )
   return {
     unrecognizedOptions,
-    config: mergeConfig(intermediateConfig, props.config ?? {}),
+    config: mergeConfig(intermediateConfig, config.value ?? {}),
```

Comment:
This change looks unnecessary. Is it?

Reply:
is `props.config` already reactive on its own?

Reply:
Yes, config as a whole is a reactive object. Accessing its property is equivalent to accessing ref's `value`.